src/Orleans.Core.Abstractions/Runtime/IGrainTimer.cs

```diff
@@ -3,12 +3,14 @@
 
 namespace Orleans.Runtime
 {
-    internal interface IGrainTimer : IDisposable
+    public interface IGrainTimer : IDisposable
     {
         void Start();
 
         void Stop();
 
         Task GetCurrentlyExecutingTickTask();
```

Comment:
I think the above three methods are less relevant to callers and they shouldn't use them. Therefore, I wonder if the existing `IGrainTimer` should be renamed to `IInternalGrainTimer` and the `Change` method added to a new `IGrainTimer`

Reply:
Yes, I'll change it ASAP.